libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -49,7 +52,24 @@ class LnurlPayInfoJob(
         var request: LnurlInfoRequest? = null
         try {
             request = Json.decodeFromString(LnurlInfoRequest.serializer(), payload)
-            val nodeState = breezSDK.nodeInfo()
+            // Get the fee parameters offered by the LSP for opening a new channel
+            val ofp =
+                breezSDK.openChannelFee(OpenChannelFeeRequest(amountMsat = null)).feeParams
+            // Calculate maximum receivable amount that falls within fee limits(in millisatoshis)
+            val feeLimitMsats: ULong = config.autoChannelSetupFeeLimitMsats
+            val nodeInfo = breezSDK.nodeInfo()
+            val maxReceivableMsatsThatFallsWithinFeeLimits = minOf(
+                nodeInfo.maxReceivableMsat,
+                feeLimitMsats / (ofp.proportional / 1000000u),
+            )
+            // Calculate maximum sendable amount(in millisatoshis)
+            val maxSendable = maxOf(
+                nodeInfo.inboundLiquidityMsats,
+                maxReceivableMsatsThatFallsWithinFeeLimits,
+            )
+            // Get the minimum sendable amount(in millisatoshis), can not be less than 1 sats
+            val minSendable: ULong =
+                if (nodeInfo.inboundLiquidityMsats == 0UL) ofp.minMsat else 1000UL
```

Comment:
Sorry for the back-and-forth (just thinking it through in my head), I think it makes more sense if the inbound is less than 1_000 then use the ofp.minMsat. 

My reasoning is if the available inbound is say 500, then the minSendable is set to 1_000. Receiving 1_000 would fail (not enough inbound / new channel needed) because it doesn't meet the minMsat requirement of the fee params. What do you think?
```suggestion
                if (nodeInfo.inboundLiquidityMsats < 1000UL) ofp.minMsat else 1000UL
```

Reply:
Makes sense👍
I still feel like we're still missing a couple edge cases😅

Reply:
Yeah... 
maybe if the maxSendable (calculated from the feeLimit) is less than the minSendable (fee param minMsat), could that happen? I guess if the provided feeLimit is too small it could. Then throw an error

Reply:
Good points.

Use minMsat from fee params if inbound liquidity is less than 1000 msats 0458995402d3ed884a8225c8b0fbb9ccee980fb2

Throw an exceptions is minSendable is greater than maxSendable a9e4f06bc1c5d2f053b1f505a6fbee9a474420b1